portal/README.md

```diff
@@ -102,3 +102,4 @@ BSS could be seperated from Portal and the Portal UI's only purpose is to login
 3. Product pages served (/bss-ui or /gef-ui)
 
 # DTFS
+Trigger 12312312312312312
```

Comment:
we can remove this and replace it with a blank space if needed

Reply:
Removed